checkClock/app/controllers/home_controller.rb

```diff
@@ -61,4 +61,11 @@ def out_employee
     end
   end
 
+  def getBranches 
```

Comment:
ruby codes uses snake_case convention, not camel case 😄 
While it works, it's better to follow the conventions